cvat-core/src/server-proxy.js

```diff
@@ -812,6 +812,16 @@
                 }
             }
 
+            async function getAllowedApps() {
+                const {backendAPI} = config;
+                try {
+                    const response = await Axios.get(`${backendAPI}/apps/`);
```

Comment:
Please, add a proxy setting to the request to be consistent (look at other requests)